data/files/source.txt

```diff
@@ -1,4 +1,2 @@
-EXPLAIN
-SELECT x.* FROM SRC x;
-
-SELECT x.* FROM SRC x;
+SELECT 'Executed SQL query 1 using SOURCE command';
+SELECT 'Executed SQL query 2 using SOURCE command';
```

Comment:
nit: It probably doesn't matter, but earlier we were testing EXPLAIN too. If you think it is needed, please add an EXPLAIN statement too.

Reply:
You are right. It doesn't matter. As @zabetak wrote: 
The goal of the test source.q is to test if the command source works at all.